draft-ietf-jsonpath-base.md

```diff
@@ -183,12 +183,14 @@ Value:
   composed of components such as structured values, namely JSON objects and arrays, and
   primitive data, namely numbers and text strings as well as the special
   values null, true, and false.
+  {{-json}} focuses on the textual representation of JSON values and
+  doesn't define the value abstraction assumed here.
```

Comment:
```suggestion
  doesn't fully define the value abstraction assumed here.
```

Section 1 and 3 together do define a value abstraction, only just not very well.